src/pages/profiles/[userId].tsx

```diff
@@ -285,6 +299,14 @@ function Picture({ userId, profile, updateProfile, SaveButton }: {
             <HStack><MdCloudUpload /><Text>上传照片</Text></HStack>
           }
         </Link>
+        {profile.照片链接 && 
+          <HStack onClick={()=>setCropImageModalOpen(!isCropImageModalOpen)}>
```

Comment:
`onClick={() => setIsCropping(true)}`. also notice necessary spaces. Use Cursor to correct missing or extra spaces and other punctuations.